etc/sysconfig/init

```diff
@@ -0,0 +1,8 @@
+# Common settings for normal and recovery init.
+
+# Amount of space to keep unallocated for refilling root or home later on.
+# Please keep this multiple of 500MB
+RESERVE_MB=0
```

Comment:
This reserved space here means space that is not allocated for the LVM. It is not the filesystem reserved space what you think. If I interpret your comment right.